src/invoices/InvoiceDetails/utils.js

```diff
@@ -128,6 +128,14 @@ export const showUpdateInvoiceError = async (
       }
       break;
     }
+    case ERROR_CODES.outdatedFundIdInEncumbrance: {
```

Comment:
Since it's a simple error without any values, maybe it makes sense to place this case a bit higher, like:
```
58 case ERROR_CODES.lockCalculatedTotalsMismatch:
59 case ERROR_CODES.outdatedFundIdInEncumbrance:
60 case ERROR_CODES.multipleFiscalYears: {
61  showCallout({
62    messageId: `ui-invoice.invoice.actions.approve.error.${ERROR_CODES[code]}`,
63      type: 'error',
64    });
65    break;
66  }
```